Vagrantfile

```diff
@@ -25,7 +25,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.define "sensu-server", primary: true, autostart: true do |server|
     server.vm.box = "centos/7"
     server.vm.hostname = 'sensu-server.example.com'
-    server.vm.network :private_network, ip: "192.168.56.10"
+    server.vm.network :private_network, ip: ENV['VAGRANT_SENSU_SERVER_IP'] || '192.168.56.10'
```

Comment:
Sure, thing, documentation added to the top of the Vagrantfile.  I also renamed the variables and consolidated to `ENV['ALTERNATE_IP']` for clarity.